terraform/backend/module/alb.tf

```diff
@@ -1,12 +1,41 @@
+data "aws_region" "current" {}
+data "aws_caller_identity" "current" {}
+
+resource "aws_s3_bucket" "log-bucket" {
+  bucket_prefix = "${var.env}-${var.app}-external-alb-logs"
+}
+
+data "template_file" "log-bucket-policy" {
+  template = "${file("${path.module}/log-bucket-policy.json.tpl")}"
```

Comment:
These vars look like it was made/linted with tf11

Reply:
If looking to add in-line policy yo the resource, it should resolve the TF linting issue. 
But yeah, my env is set on 11 =\
Nonetheless, feel free to pull & commit proper TF 12 linting and move the bucket policy to the same alb.tf resource.